2008-01-25  Jakub Jelinek  <jakub@redhat.com>

	* lang.c (java_classify_record): Revert 2007-12-20 change.

--- gcc/java/lang.c	2007-12-27 09:09:49.000000000 +0100
+++ gcc/java/lang.c	2008-01-25 17:43:57.000000000 +0100
@@ -965,9 +965,7 @@ java_classify_record (tree type)
   if (! CLASS_P (type))
     return RECORD_IS_STRUCT;
 
-  /* ??? GDB does not support DW_TAG_interface_type as of December,
-     2007.  Re-enable this at a later time.  */
-  if (0 && CLASS_INTERFACE (TYPE_NAME (type)))
+  if (CLASS_INTERFACE (TYPE_NAME (type)))
     return RECORD_IS_INTERFACE;
 
   return RECORD_IS_CLASS;
